tests/candev/main.c

```diff
@@ -222,6 +222,18 @@ int main(void)
 
     candev->driver->init(candev);
 
+#if defined(CAN_LOOPBACK_MODE)
+    puts("Switching to loopback mode");
+    /* set to loopback test mode */
+    canopt_state_t mode = CANOPT_STATE_LOOPBACK;
+    candev->driver->set(candev, CANOPT_STATE, &mode, sizeof(mode));
+
+    /* dont care, rev all msg id */
+    struct can_filter filter;
+    filter.can_mask = 0;
+    candev->driver->set_filter(candev, &filter);
+#endif
```

Comment:
I would change this to some kind of configuration flag something like `CONFIG_USE_LOOPBACK_MODE` and then apply:

```suggestion
if (IS_ACTIVE(CONFIG_USE_LOOPBACK_MODE)) {
    puts("Switching to loopback mode");
    /* set to loopback test mode */
    canopt_state_t mode = CANOPT_STATE_LOOPBACK;
    candev->driver->set(candev, CANOPT_STATE, &mode, sizeof(mode));

    /* dont care, rev all msg id */
    struct can_filter filter;
    filter.can_mask = 0;
    candev->driver->set_filter(candev, &filter);
}
```

Also did you mean "recv" instead of rev?

Reply:
good suggestion.